docs/docs/governance/unlock-dao-tokens.mdx

```diff
@@ -72,50 +49,164 @@ import { networks } from '@unlock-protocol/networks'
 
-- Higher value transactions yield a larger amount of UP tokens, everything else being equal,
-- Earlier transactions yield a larger amount of UP tokens, everything else being equal,
-- The cost of buying UP on a decentralized exchange is always lower than the cost of sending "fake" transactions to the network in order to collect UP.
+- Higher value transactions yield a larger amount of UP token rewards, everything else being equal; and,
+- Earlier transactions yield a larger amount of UP tokens, everything else being equal; and,
+- The cost of buying UP on a decentralized exchange is always lower than the cost of sending "fake" transactions to the network in order to collect UP
 
 Another way to earn UP and join the DAO is to [apply for grants](./grants-bounties).
 
 ## Protocol Fee
 
-As of Unlock version 13, the protocol includes a "fee switch" and the fee was initially set to be 0%. This value has been changed by [DAO members](./unlock-dao) through an [onchain proposal](https://www.tally.xyz/gov/unlock/proposal/72110981722145472193202862106710876303006911465748904597113931014096574827698) and approved through votes to 1% in May 2024.
+Unlock Protocol includes a "fee switch” that is currently set to 1%. This value was set by the Unlock [DAO](./unlock-dao) through an [onchain proposal](https://www.tally.xyz/gov/unlock/proposal/72110981722145472193202862106710876303006911465748904597113931014096574827698) and approved in May 2024.
 
-If the fee amount is larger than 0%, on every transaction for which there is a payment ([`purchase`](/core-protocol/smart-contracts-api/PublicLock#purchase), [`extend`](/core-protocol/smart-contracts-api/PublicLock#extend) or [`renewMembershipFor`](/core-protocol/smart-contracts-api/PublicLock#renewmembershipfor)), the lock contract will send a portion of the payment to the Unlock contract.
+When the fee switch is set to any value larger than 0%, on every transaction for which there is a payment ([`purchase`](/core-protocol/smart-contracts-api/PublicLock#purchase), [`extend`](/core-protocol/smart-contracts-api/PublicLock#extend) or [`renewMembershipFor`](/core-protocol/smart-contracts-api/PublicLock#renewmembershipfor)), the lock contract will send a portion of the payment to the Unlock contract on that network. These fees are used to help sustain the decentralized protocol.
 
 ### Swap and Burn
 
-At that point, the collected fees can be "swapped and burned" by calling the `swapAndBurn` function on the Unlock contract. This function will swap the collected fees for UP (or UDT) tokens on a decentralized exchange and then burn them.
+Current, the protocol fees collected by way of the fee switch can be "swapped and burned" by calling the `swapAndBurn` function on the Unlock contract. This function will swap the collected fees for UP (or UDT) tokens on a decentralized exchange and then burn them.
+
+## Bridging UDT from Mainnet to Base and Exchanging the UDT for the Unlock Protocol UP Token
```

Comment:
This looks a bit strange.

<img width="848" alt="Screenshot 2024-09-16 at 4 45 06 PM" src="https://github.com/user-attachments/assets/59b8147e-4d8d-4f99-99ab-2616dec71872">

Maybe we need to make this title shorter?